lms/static/sass/course/courseware/_courseware_search.scss

```diff
@@ -61,7 +62,7 @@
 
   .search-results-item {
     position: relative;
-    border-bottom: 1px solid lighten($border-color, 10%);
+    border-bottom: 1px solid $border-color-1;
```

Comment:
@dsego - my guess is that gray-l4 is what you are looking for. I would try that.